packages/contracts/docs/osx/02-how-to-guides/01-dao/01-best-practices.md

```diff
@@ -2,23 +2,21 @@
 title: Best Practices
 ---
 
-## Advices for Operating your DAO
+## Some Advice When Operating your DAO
 
 ### DOs 👌
 
-- Make sure that at least one address (typically a governance plugin) has `EXECUTE_PERMISSION_ID` permission so that it can execute on the DAO.
-- Check every proposal asking to install, update, or uninstall a plugin with utmost care and review
+- Make sure that at least one address (typically a governance plugin) has `EXECUTE_PERMISSION_ID` permission so that something can be execute on behalf of the DAO.
+- Check every proposal asking to install, update, or uninstall a plugin with utmost care and review. Installation means granting an external contract permissions, so you want to be extra careful about:
```

Comment:
It is not only about contracts. Addresses in general get permissions and these can also be accounts.
It is also not only about "Installation" but for every setup type (Installation, Update, Uninstallation) there must be a very careful review of the items below.

I would leave the sentence as it was